redis/src/cluster_async/routing.rs

```diff
@@ -144,8 +144,10 @@ mod pipeline_routing_tests {
         pipeline
             .get("foo") // route to replica of slot 12182
             .add_command(cmd("FLUSHALL")) // route to all masters
-            .add_command(cmd("EVAL"))// route randomly
-            .cmd("CONFIG").arg("GET").arg("timeout") // unkeyed command
+            .add_command(cmd("EVAL")) // route randomly
+            .cmd("CONFIG")
```

Comment:
adding a command per-line in pipelines is clearer than this, which doesn't separate commands from args. Is there a way to prevent this?

Reply:
Of course, you can add parameters to the config. I will finish this PR.